Widgets 2.0 (#4028)
DataDog/documentation: content/en/graphing/widgets/iframe.md

---
title: Iframe Widget
kind: documentation
description: Inlude an Iframe in your Datadog dashboards.
further_reading:
- link: "graphing/dashboards/screenboard/"
  tag: "Documentation"
  text: "Screenboard"
---

> The Iframe widget allows you to embed a portion of any other web page on your dashboard.

Reviewer: ```suggestion
The iframe widget allows you to embed a portion of any other web page on your dashboard.
```

## Setup

{{< img src="graphing/widgets/iframe/iframe_setup.png" alt="Iframe setup" responsive="true" style="width:80%;">}}

> Enter the URL of the Iframe you want to display. You should use an HTTPS URL, or you may have to configure your browser to allow non-secure content.

Reviewer: ```suggestion
Enter the URL of the page you want to display inside the iframe. If you do not use an HTTPS URL, you may have to configure your browser to allow non-secure content.
```
